web/src/main/bvmui/src/components/Ledger.vue

```diff
@@ -13,14 +13,25 @@
             <span>Replication {{ ledger.writeQuorumSize }}</span>
         </div>
         <div class="bvm-tile__row">
-            <span>Age {{ ledger.age }} hours</span>
+            <span>Age {{ computedAge }}</span>
         </div>
     </div>
 </template>
 <script>
 export default {
     props: {
         ledger: Object
+    },
+    computed: {
+        computedAge() {
```

Comment:
Can we add some unit tests for this function?

Reply:
Done, started the tests in the frontend